src/NzbDrone.Core/Download/Clients/Porla/Porla.cs

```diff
@@ -0,0 +1,239 @@
+
+            var status = new DownloadClientInfo
+            {
+                IsLocalhost = Settings.Host == "127.0.0.1" || Settings.Host == "localhost",
+                RemovesCompletedDownloads = false, // should be determined through Flags From  somewhere? I think session?
+            };
+
+            // the default directory value is stored in the presets values... i think?
+            // but will not always be active if not set
+
+            // destDir = new OsPath(preset.GetValueOrDefault("save_path") as string);
+            // if (destDir.IsNotNullOrEmpty)
+            // {
+            //    status.OutputRootFolders = new List<OsPath> { _remotePathMappingService.RemapRemoteToLocal(Settings.Host, destDir) };
+            // }
+
+            return status;
+        }
+
+        private static IList<string> ConvertRemoteEpisodeToTags(RemoteEpisode remoteEpisode)
+        {
+            var tags = new List<string>
+            {
+                $"starr.series={remoteEpisode.Series.CleanTitle}",
+                $"starr.season={remoteEpisode.MappedSeasonNumber}",
+                $"starr.tvdbid={remoteEpisode.Series.TvdbId}"
+            };
+            return tags;
+        }
+
+        protected override string AddFromMagnetLink(RemoteEpisode remoteEpisode, string hash, string magnetLink)
+        {
+            var tags = Settings.SeriesTag ? ConvertRemoteEpisodeToTags(remoteEpisode) : null;
+            var torrent = _proxy.AddMagnetTorrent(Settings, magnetLink, tags);
+            return torrent.InfoHash.Hash;
+        }
+
+        protected override string AddFromTorrentFile(RemoteEpisode remoteEpisode, string hash, string filename, byte[] fileContent)
+        {
+            var tags = Settings.SeriesTag ? ConvertRemoteEpisodeToTags(remoteEpisode) : null;
+            var torrent = _proxy.AddTorrentFile(Settings, fileContent, tags);
+            return torrent.InfoHash.Hash;
+        }
+
+        protected override void Test(List<ValidationFailure> failures)
+        {
+            failures.AddIfNotNull(TestConnection());
+            if (failures.HasErrors())
+            {
+                return;
+            }
+
+            failures.AddIfNotNull(TestGetTorrents());
+        }
+
+        private ValidationFailure TestConnection()
+        {
+            try
+            {
+                var sysVers = _proxy.GetSysVersion(Settings);
+                var version = new Version(sysVers.Porla.Version);
+
+                if (version < new Version("0.37.0"))
+                {
+                    // return new ValidationFailure(string.Empty, _localizationService.GetLocalizedString("DownloadClientValidationErrorVersion",
+                    //        new Dictionary<string, object> { { "clientName", Name }, { "requiredVersion", "0.37.0" }, { "reportedVersion", version } }));
+                    _logger.Warn($"Porla: Unsure if your version is forwards compatible: {version}");
```

Comment:
Not sure why this is needed if you're also checking for < 1.0

Reply:
Checking forward and backward and the exact version we know is fine.
Maybe the future breaks, maybe we are incompatible with the past.
Wanted to get the blocks in so it's easy to change the numbers when that happens.

Reply:
Oh the < 1.0 check!

That is just a general warning in the logs for others.
The checks above it are doing the actual comparability stuff